docs/rustdoc/centralized_telescope/main.md

```diff
@@ -1,3 +1,55 @@
 # Telescope - Construction with Bounded DFS
-This is the main page of centralized documentation.
-It will include a generic explanation of the protocol.
+
+In scenarios where $n_p$ is small, the parameters given for prehashed construction (relative to the security parameter, $\lambda$) are not optimal, resulting in a reduced probability of constructing a valid proof in a single attempt.
+For large $n_p$, the rapid growth in potential proof tuples ensures valid ones can be found efficiently, allowing the prehashed construction to work seamlessly.
+In contrast, small $n_p$ limits the search space, making the previous parameters inadequate.
+To address this, the **construction with bounded DFS** expands on the prehashed version with *retries*, *randomized indexing*, and *bounding* the DFS.
```

Comment:
what is randomized indexing?

Reply:
Prehashing with retry counter $r$.

Reply:
Something like `prehashing randomization` mightbe clearer then?

Reply:
I would omit it altogether. "Retries" already covers it, the counter $r$ is an implementation detail.